index.html

```diff
@@ -0,0 +1,137 @@
+            ipsam iste itaque nihil architecto, volu tatem sed facilis porro
+            maiores, rerum vitae.
+          </p>
+        </div>
+        <p class="date">05(FRI)-07(SUN) DEC 2021</p>
+        <p class="at">@ YouTube: Microverse Official Channel</p>
+      </section>
+      <section class="program">
+        <h2>Main Program</h2>
+        <div class="program-wrap">
+          <div class="program-item">
+            <img src="assets/img/icon1.png" alt="" />
+            <h3>Live Music</h3>
+            <p>
+              More than 50 artists amet consectetur adipisicing elit. Repudiandae
+              harum nisi, aliquid.
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon2.png" alt="" />
+            <h3>Visual arts</h3>
+            <p>
+              Trippy visuals from consectetur adipisicing elit. Repudiandae harum
+              nisi.
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon3.png" alt="" />
+            <h3>Interviews</h3>
+            <p>
+              On stage with amet consectetur. Repudiandae harum nisi, aliquid
+              adipisicing elit.
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon4.png" alt="" />
+            <h3>20 Deejays</h3>
+            <p>
+              Turn on the volumen consectetur adipisicing elit. Repudiandae harum
+              nisi, aliquid sound!
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon5.png" alt="" />
+            <h3>Live Streaming</h3>
+            <p>
+              Watch it live sit adipisicing elit. Repudiandae harum nisi, aliquida
+              met consectetur.
+            </p>
+          </div>
+        </div>
+        <div class="program-link">
+          <a href="#" class="program-link">See the whole program</a>
+        </div>
+      </section>
+      <section class="featured" id="featured"></section>
+      <section class="partners">
+        <h2>Partners</h2>
+        <div class="partners-wrap">
+          <img src="assets/img/logo.png" alt="" />
+          <img src="assets/img/logo2.png" alt="" />
+          <img src="assets/img/logo3.png" alt="" />
+          <img src="assets/img/logo4.png" alt="" />
+          <img src="assets/img/logo5.png" alt="" />
+        </div>
+      </section>
+    </div>
+    <div>
+      <div class="footer-image">
+        <img src="assets/img/logo.png" alt="" />
+      </div>
+      <div class="footer-text">
+        <p class="bold">
+          Microverse Project HTML CSS JS 2021. All rights reserved.
+        </p>
+        <p>
+          Thanks for supporting this concert. Repudiandae harum nisi,
+          consectetur adipisicing elitl lorem ipsum. Lorem ipsum dolor sit amet
+          consectetur adipisicing elit.
+        </p>
+      </div>
+    </div>
```

Comment:
[OPTIONAL] For this part I would strongly recommend using a `footer` tag in order to have a well-defined HTML structure.